shoop/admin/modules/permission_groups/views/edit.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+import ast
+
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group as PermissionGroup
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _
+
+from shoop.admin.forms.widgets import Select2Multiple
+from shoop.admin.module_registry import get_modules
+from shoop.admin.permissions import get_permission_object_from_string
+from shoop.admin.utils.views import CreateOrUpdateView
+
+
+class PermissionGroupForm(forms.ModelForm):
+    class Meta:
+        model = PermissionGroup
+        exclude = ("permissions",)
+
+    def __init__(self, *args, **kwargs):
+        super(PermissionGroupForm, self).__init__(*args, **kwargs)
+        initial_permissions = self._get_initial_permissions()
+        self.fields["modules"] = forms.MultipleChoiceField(
+            choices=sorted(self._get_module_choices()),
+            initial=self._get_enabled_modules(initial_permissions),
```

Comment:
enabled_modules in this context means that they're enabled based on the groups initial permissions. However, I did change it so that the initial form is empty. Before it showed all modules that didn't have any permission requirements. Now, those are hidden until after the initial save.